fe2-android/app/src/test/unit/java/com/github/dedis/popstellar/utility/handler/ConsensusHandlerTest.java

```diff
@@ -89,8 +89,13 @@ public class ConsensusHandlerTest {
 
   @Before
   public void setup()
-      throws GeneralSecurityException, DataHandlingException, IOException, UnknownLaoException,
-          UnknownRollCallException, UnknownElectionException, NoRollCallException,
+      throws GeneralSecurityException,
```

Comment:
I think this is again a formatting error

Reply:
Same as above, double check and that's the output of the formatter as of now

Reply:
I just tried it and I get the same results. I really seems like they changed the formatting.